pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,123 @@
+"""DP computing of count, sum, mean, variance."""
+
+import numpy as np
+import pipeline_dp
+
+from dataclasses import dataclass
+
+
+@dataclass
+class MeanVarParams:
+	"""Manages the parameters used for computing the mean/variance."""
+	eps: float
+	delta: float
+	low: float
+	high: float
+	max_partitions_contributed: int
+	max_contributions_per_partition: int
+	noise_kind: pipeline_dp.NoiseKind  # Laplace or Gaussian
+
+
+def _apply_laplace_mechanism(value, dp_params):
```

Comment:
Sensitivity also depends on type of aggregation. Pease replace dp_params -> l1_sensitivity and compute sensitivity in compute_dp_count, compute_dp_sum.